saul-examples/src/main/scala/edu/illinois/cs/cogcomp/saulexamples/nlp/EntityRelation/EntityRelationApp.scala

```diff
@@ -112,8 +112,8 @@ object EntityRelationApp extends Logging {
     val testTokens = tokens.getTestingInstances.toSet.toList
 
     // load pre-trained independent models
-    ClassifierUtils.LoadClassifier(jarModelPath, PersonClassifier, OrganizationClassifier, LocationClassifier,
-      WorksForClassifier, LivesInClassifier, LocatedInClassifier, OrgBasedInClassifier)
+    //    ClassifierUtils.LoadClassifier(jarModelPath, PersonClassifier, OrganizationClassifier, LocationClassifier,
+    //      WorksForClassifier, LivesInClassifier, LocatedInClassifier, OrgBasedInClassifier)
```

Comment:
yes, just to show that we do not need to pre-train. Although I believe the performance drops without pre-training.